services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.dateparse import parse_datetime
+
+from db.models import Ticket, Order
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str | None = None
+) -> Order:
+
+    user = get_user_model().objects.get(username=username)
```

Comment:
shouldn't this be as well under `atomic` decorator?

Reply:
Lol this actually is a read operation that wouldn't need a rollback in case if something happens :)
But it is actually better to wrap the entire function in transaction